.github/CODEOWNERS

```diff
@@ -11,3 +11,9 @@ spec/support/*/vaos/                 @department-of-veterans-affairs/vfs-vaos @d
 
 lib/bgs @department-of-veterans-affairs/vfs-ebenefits @department-of-veterans-affairs/backend-review-group
 spec/bgs @department-of-veterans-affairs/vfs-ebenefits @department-of-veterans-affairs/backend-review-group
+
+modules/mobile/                      @department-of-veterans-affairs/mobile-api-team @department-of-veterans-affairs/backend-review-group
+lib/iam_ssoe_oauth                   @department-of-veterans-affairs/mobile-api-team @department-of-veterans-affairs/backend-review-group
```

Comment:
anything outside of `modules/mobile` is shared code that VSP has to support, so making mobile-api-team code owners outside that directory isn't going to work.  Moving code that doesn't need to be shared with the platform into `modules/mobile` would be a good approach to own some of that stuff.

Reply:
Did you want to own `spec/support/*/mobile/`?  Originally you were going to copy vaos and thought that was a good approach.  But I'll approve it without that if you want

Reply:
@erluti we were going to try to have our spec support files also in the engine (still researching how to do this). But ya for now I'll add the mobile support dir back in.